enzy_htp/structure/residue.py

```diff
@@ -270,6 +275,14 @@ def n_side_residue(self)-> Union["Residue", None]:
                                 " Something wrong in your residue indexing. "
                                 "It could be your index is not continous")
         return result
+    
+    def find_idxes_atom_list(self, atom_idx_list: int) -> List[Atom]:
+        """find atom base on its idx. return a list reference of the atoms."""
+        result = []
+        atom_mapper = self.atom_idx_mapper
+        for idx in atom_idx_list:
+            result.append(atom_mapper[idx])
+        return result
     #endregion
```

Comment:
Why do we need this?

Reply:
Not sure...must've added it while debugging/adding the convert_to_structure function

Reply:
Oh okay, it is needed in the creation of a structure selection